src/test/java/guitests/DialogMessageTests.java

```diff
@@ -38,4 +38,12 @@ public void showYesNoWarningDialogTest() throws ExecutionException, InterruptedE
         click("nO");
         assertEquals(false, noTask.get());
     }
+
+    @Test
+    public void showInformationDialogTest() {
+        Platform.runLater(() -> DialogMessage.showInformationDialog("Information", "Test Information Dialog"));
+        sleep(1000);
```

Comment:
So we wait until `hasText` is true then assert the `hasText` again? But implementing the wait will involve `sleep` as well, yes? And currently there is no method to `waitUntilNodeAppears(Matcher<>)`. Should I make one?

If this gets on, should I change the two tests in the same file as this to use that wait as well?